src/Exercism.Analyzers.CSharp/Analyzers/Leap/LeapSyntaxFactory.cs

```diff
@@ -22,6 +22,19 @@ internal static class LeapSyntaxFactory
                 LeapModuloExpression(leapSolution, 400),
                 NumericLiteralExpression(0)));
 
+        public static BinaryExpressionSyntax LeapMinimumNumberOfChecksWithoutParenthesesBinaryExpressionReversed(LeapSolution leapSolution) =>
+            LogicalOrExpression(
+                EqualsExpression(
```

Comment:
The `EqualsExpression` is shared with the method above. Could you extract it to a helper method?